js/login.js

```diff
@@ -0,0 +1,41 @@
+/* variables */
+
+let logIn = document.getElementById("login-btn");
+let loginForm = document.querySelector(".login-form");
```

Comment:
```suggestion
const loginForm = document.querySelector(".login-form");
```
Instead of using let for variables that won't be reassigned, use const